Machine Learning/Hands-on Implementation using sklearn/Linear Regression/readme.md

```diff
@@ -0,0 +1,13 @@
+# Linear Regression with scikit-learn in Python
+
+Implemented Using: Jupyter Notebook
```

Comment:
I don't feel there would be a need to add Implemented Using could you remove this?

Reply:
Sure I'll remove this.